tests/test_publisher.py

```diff
@@ -0,0 +1,79 @@
+import os
+import sys
+from unittest.mock import call
+
+import pytest
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
```

Comment:
please, don't do that. you can use pytest outside tests folder just runing pytest in the project root